nxdrive/report.py

```diff
@@ -111,10 +112,8 @@ def export_logs(lines: int = constants.MAX_LOG_DISPLAYED) -> Iterator[bytes]:
             try:
                 line = handler.format(record)
             except:
-                try:
+                with suppress(AttributeError):
```

Comment:
Hm same here. `UnicodeDecodeError` may be possible too. `Exception` is too broad you think?